polymesh_fr.properties

```diff
@@ -347,20 +347,36 @@ menu.preferences=Pr
 menu.showSelectionOnPreview=Montrer la s�lection sur la pr�visualisation
 menu.liveUpdate=Suivi des modifications en direct
 menu.boldEdges=Ar�tes grasses
+#menu.fitToAll=Fit to All
+#menu.fitToSelection=Fit to Selection
```

Comment:
Why are we adding enlish placeholders to the translation file?

Reply:
Doesn't the question pretty much answer itself? ;)

The plan was, that you have the placeholders for new things, that need translating and they contain the default language text right where you will need the translation.

This way, who ever does any translating in future, does not have to compare the outdated translation file to the new default file and try to find, what possibly would have changed.  Nobody knows better, where the new texts are than who ever put them there and it is a much smaller job to add the placeholders to all of the files right away than to try to find changes afterwards. 

I marked them comments for clarity, so it's easier to spot them and also of course to keep the "wrong" language bits out of the actual translation.

I'd suggest to take this as future practice with core AoI too, though currently we'd need some more powerful tools to process all the files in one step to the same level. Some parts of the default have been reordered, some translations have additional comment lines ... but I'd expect it to facilitate considerably the work of keeping them up to date. _(... Just if somebody would, but I think it'd be a good approach to assume just that.)_

Reply:
Hmm... I see where you're coming from. I'm not sure if that's the way to go, though.

I'm currently looking at possibly building a plugin that would allow for live editing of translations in the application. I'm concerned in the meantime that all of the placeholders would possibly be noise...

Reply:
Basically, I think to leave these in for now. We'll see where a translation plugin goes, and then go from there...